tools/rostopic/src/rostopic/__init__.py

```diff
@@ -207,7 +207,107 @@ def _rostopic_hz(topic, window_size=-1, filter_expr=None):
     while not rospy.is_shutdown():
         _sleep(1.0)
         rt.print_hz()
-    
+
+
+class ROSTopicDelay(object):
+
+    def __init__(self, window_size):
+        import threading
```

Comment:
Please move the imports to the top of the module (in alphabetical order following PEP8).